package.json

```diff
@@ -51,7 +52,7 @@
         "@types/chai-as-promised": "^7.1.4",
         "@types/chai-spies": "^1.0.4",
         "@types/mocha": "^9.0.0",
-        "@types/node": "^18.19.71",
+        "@types/node": "22.13.1",
```

Comment:
Is this change necessary? @relu91 ?

Reply:
we still support Node 18, so the maximum upgrade that we can do here is to move the minor. See https://github.com/eclipse-thingweb/node-wot/blob/master/.github/workflows/ci.yaml#L22

Reply:
This should not cause any problem and will put you ahead of the curven but we can downgrade,

Reply:
I noticed that the CI runs on Node 18.x and 20.x were fine. @relu91 any opinion?

Reply:
> I noticed that the CI runs on Node 18.x and 20.x were fine. @relu91 any opinion?

They might run fine now, but we would notice only late if Node 22.x introduces any breaking changes in the API. This is something very rare but it can happen. Since we don't need any new feature of Node 22 (yet), I'd prefer to stick with 18.x. Plus the scope of this PR is something else and this change is not strictly related.